### PR TITLE
Create validators for each model.

### PR DESCRIPTION
**Pull request by laurandarius**

### CODE REVIEW

**Comment by laurandarius** [comment]
[Create validators for each model.](https://trello.com/c/vtjttmDG/22-create-validators-for-each-model)

File: app/services/companyReview.js [hunk 1]
```diff
@@ -1,47 +1,35 @@
...
-      error.throwValidationError('Invalid company review format.');
-    }
-
-    const companyReviewExists = await companyReviewRepository.getByPropsNonParanoid(companyReview);
-    if (companyReviewExists) {
-      error.throwValidationError('Company review already exists.');
-    }
-
-    await companyService.get(companyId);
+    await companyReviewValidator.validateCreate(companyReview, user, companyId);
```
Comment (lgnstn) [inline app/services/companyReview.js]: Why didn't you use the same way of using the validator? With `this.validator.validate...`?

File: app/validators/crudValidator.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+const error = require('../services/error');
+
+module.exports = class CrudValidator {
+  constructor(props) {
+    this.modelName = props.modelName;
+    this.repository = require(`../repositories/${props.repositoryName}`);
+  }
+
+  async validateCreate(record) {
+    if (record.id !== undefined) {
```
Comment (lgnstn) [inline app/validators/crudValidator.js]: Do you use this code?

**Review by lgnstn — CHANGES_REQUESTED** [review]

**Comment by laurandarius** [comment]
@lgnstn updated.

**Review by lgnstn — APPROVED** [review]

**Pull request merged**